### PR TITLE
Set smallest circle size to 1

### PR DESCRIPTION
**Pull request by domoritz**
image

### CODE REVIEW

**Comment by domoritz** [comment]
Hmm, I wonder whether we should do this only for marks with a stroke. If you have a fill, size 0 should really not be visible.

File: src/compile/scale/range.ts [hunk 1]
```diff
@@ -214,6 +214,10 @@ export function defaultRange(
 
 function sizeRangeMin(mark: Mark, zero: boolean, config: Config) {
   if (zero) {
+    if (util.contains(['point', 'square', 'circle'], mark)) {
+      // the smallest size for points should not be zero as the marks become invisible
```
Comment (kanitw) [inline src/compile/scale/range.ts]: But this also make scale incorrect. 

I'd say, if you have "zero" value, better not use size.

Comment (kanitw) [reply src/compile/scale/range.ts]: Maybe 0.01 or 0.1 is palatable

Comment (domoritz) [reply src/compile/scale/range.ts]: Try it out, you still have white inside the circle but it looks worse.

Comment (kanitw) [reply src/compile/scale/range.ts]: Already tried and it's terrible.  In general, using size to encode field with zero values is a bad idea anyway, so I don't think we should make size less accurate for this bad use case.

Comment (domoritz) [reply src/compile/scale/range.ts]: The change in accuracy is small but https://github.com/vega/vega-lite/issues/3488 seems problematic. If you disagree, we should close the issue and this PR.

Comment (kanitw) [reply src/compile/scale/range.ts]: Let's try production rule? https://github.com/vega/vega-lite/issues/3488#issuecomment-383388466

**Pull request closed without merging**